libraries/networking/src/LimitedNodeList.cpp

```diff
@@ -428,6 +428,10 @@ void LimitedNodeList::handleNodeKill(const SharedNodePointer& node) {
     qCDebug(networking) << "Killed" << *node;
     node->stopPingTimer();
     emit nodeKilled(node);
+    
+    if (auto activeSocket = node->getActiveSocket()) {
```

Comment:
I just found out you can do that!!!
Apparently it's available since C++98, Mind = Blown.